test/AutoRest.TestServer.Tests/Common/StringExtensionsTests.cs

```diff
@@ -79,6 +79,24 @@ public void ValidateSplitByCamelCase(string camelCase, string[] expected)
         [TestCase("_propertyName", true, "PropertyName")]
         [TestCase("_property_name", false, "propertyName")]
         [TestCase("_property_name", true, "PropertyName")]
+        [TestCase("_OS_profile", true, "OSProfile")]
+        [TestCase("_OS_profile", false, "osProfile")]
+        [TestCase("_OS_Profile", true, "OSProfile")]
+        [TestCase("_OS_Profile", false, "osProfile")]
+        [TestCase("OS", true, "OS")]
+        [TestCase("OS", false, "os")]
+        [TestCase("DNS", true, "DNS")]
+        [TestCase("DNS", false, "dns")]
```

Comment:
Can we add cases for something like `PrimaryDNS`?

Reply:
do you expect `primaryDNS`?